common/phone/phone_location.avsc

```diff
@@ -0,0 +1,17 @@
+{
+  "namespace": "org.radarcns.phone",
+  "type": "record",
+  "name": "PhoneLocation",
+  "doc": "Data from the gps and network location providers. The latitude and longitude are stated with an unspecified reference offset and can thus be treated as relative locations. They cannot be used to infer absolute location. This means accurate distances or angles between locations cannot be calculated since those depend on the absolute location.",
```

Comment:
should we have the possibility of capturing both relative and absolute phone location? There are real senarios where one might have permission to gather this data